profiles/tests/test_saved_company.py

```diff
@@ -53,18 +52,22 @@ def test_add_non_existent_company_to_saved_authenticated(self):
             data={
                 "company_pk": 0,
             },
+            format="json",
         )
         self.assertEqual(status.HTTP_400_BAD_REQUEST, response.status_code)
+        self.assertEqual(
+            {"non_field_errors": ["Company does not exist"]},
```

Comment:
Why this response is not formatted? 

We have a function `dump()` which allows to get response content easily, do you use it?